stubs/mysqlclient/MySQLdb/times.pyi

```diff
@@ -9,18 +10,18 @@ Timestamp = datetime
 DateTimeDeltaType = timedelta
 DateTimeType = datetime
 
-def DateFromTicks(ticks): ...
-def TimeFromTicks(ticks): ...
-def TimestampFromTicks(ticks): ...
+def DateFromTicks(ticks: float | None) -> date: ...
+def TimeFromTicks(ticks: float | None) -> time: ...
+def TimestampFromTicks(ticks: float | None) -> datetime: ...
 
 format_TIME = str
 format_DATE = str
 
-def format_TIMEDELTA(v): ...
-def format_TIMESTAMP(d): ...
-def DateTime_or_None(s): ...
-def TimeDelta_or_None(s): ...
-def Time_or_None(s): ...
-def Date_or_None(s): ...
-def DateTime2literal(d, c): ...
-def DateTimeDelta2literal(d, c): ...
+def format_TIMEDELTA(v: timedelta) -> str: ...
+def format_TIMESTAMP(d: datetime) -> str: ...
+def DateTime_or_None(s: str) -> datetime | None: ...
+def TimeDelta_or_None(s: str) -> timedelta | None: ...
+def Time_or_None(s: str) -> time | None: ...
+def Date_or_None(s: str) -> date | None: ...
+def DateTime2literal(d: datetime, c: Any) -> Any: ...
+def DateTimeDelta2literal(d: datetime, c: Any) -> Any: ...
```

Comment:
As `c` is ignored, it can be literally anything. Confusingly, this is expressed using `object`, not `Any`. The latter is used as a marker for "can't be expressed with the type system". Also, both of there return `str.

```suggestion
def DateTime2literal(d: datetime, c: object) -> str: ...
def DateTimeDelta2literal(d: datetime, c: object) -> str: ...
```